modules/workflows/OrcaRecoveryWorkflow/orca_recover_workflow.asl.json

```diff
@@ -54,7 +53,7 @@
             "glacier-bucket": "${orca_default_bucket}",
```

Comment:
Is `glacier-bucket` still needed or does this go away now?

Reply:
Unless we want to force a migration, it is still needed.

Reply:
Well, we'd be removing the reference to orca_default_bucket, right? So that could break somebody's setup.

Reply:
I don't believe so, I think that may have been a placeholder. I would expect we would get the ORCA bucket name either from either the environment `ORCA_DEFAULT_BUCKET`, or the configuration `orcaDefaultBucketOverride` this third variable I feel is confusing and does not provide any benefit. Looking at the code for request_files it really isn't used except to check for it and set the value to our environment variable. Seeing that this is a part of our workflow that we distribute, I feel like we can get rid of it. Thoughts?

Reply:
Huh... So the same value was being passed in two different ways, I guess. Alright. Removed.